lib/DDG/Spice/OpenNIC.pm

```diff
@@ -0,0 +1,81 @@
+package DDG::Spice::OpenNIC;
+# ABSTRACT: Displays The nearest OpenNIC DNS servers
+
+use DDG::Spice;
+spice is_cached => 1;
+
+# Metadata
+name "OpenNIC";
+source "OpenNIC API";
+description "Search for OpenNIC DNS using some criteras";
+primary_example_queries "opennic", "neutral dns" , "censor-free dns";
+secondary_example_queries "opennic 42.42.42.42", "opennic ipv6", "opennic ipv6 42.42.42.42";
+category "cheat_sheets";
```

Comment:
If it isn't in https://github.com/duckduckgo/zeroclickinfo-goodies/tree/master/share/goodie/cheat_sheets it probably shouldn't be called a cheat sheet.